### PR TITLE
feat: Create create_settlement_report_monthly_amounts_v1.sql

### PR DESCRIPTION
**Pull request by johevemi**
# Description

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_monthly_amounts_v1.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+CREATE VIEW {SETTLEMENT_REPORT_DATABASE_NAME}.monthly_amounts_v1 as
+SELECT wr.calculation_id,
+       wr.calculation_type,
+       wr.grid_area_code,
+       wr.energy_supplier_id,
+       wr.time AS start_date_time,
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_monthly_amounts_v1.sql:6]: I think we only use start_date_time when we have arrays of data. Other places we use 'time'.

Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_monthly_amounts_v1.sql:6]: arh sorry. I see we use this name in the csv. Nevermind

Comment (johevemi) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_monthly_amounts_v1.sql:6]: and it is also called "start_date_time" the wholesale_results_v1 view

Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create_settlement_report_monthly_amounts_v1.sql:6]: The csv does not determine the naming in the views.

**Review by energinet-ajw — DISMISSED** [review]

**Review by energinet-ajw — APPROVED** [review]

**Pull request merged**